engine/env/spark/src/main/java/org/datacleaner/spark/SparkJobLifeCycleListener.java

```diff
@@ -36,21 +36,21 @@
      * Triggered when a node starts processing a task. This will be executed
      * on the nodes themselves.
      */
-    void onPartitionProcessingStart();
+    void onPartitionProcessingStart(SparkJobContext sparkJobContext);
```

Comment:
It's coming in RowProcessingFunction, but I took a look and it's not trivial because that function has two was it can be invoked (it also implements two interfaces accordingly) and the partition number is only available in one of them. So let's not care about it for now.